libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/arrow/api/MapboxRouteArrowView.kt

```diff
@@ -33,12 +35,38 @@ import com.mapbox.navigation.utils.internal.logE
  * @param options the options used for determining the rendering appearance and/or behavior.
  */
 @UiThread
-class MapboxRouteArrowView(private val options: RouteArrowOptions) {
+class MapboxRouteArrowView(
+    private var options: RouteArrowOptions,
+    private var rebuildLayersOnFirstRender: Boolean = true
```

Comment:
This same local flag and logic should be introduced to `MapboxRouteLineView`.

Reply:
My recollection of our last conversation is we aren't adding this to the route line view.  The route line layers are a lot more complicated and there's no use case for adding this feature to the route line.  The use case for this being added to the arrows was the drop-in UI wanting to change the arrow drawable under certain conditions.

Reply:
`MapboxRouteLineView` still suffers from a bug that if the options have changed, they are not re-applied on instance recreation (or first render). It's fine if we want to create separate PRs for `MapboxRouteLineView` and `MapboxRouteArrowView` but we need to address both eventually.